gcloud/bigquery/client.py

```diff
@@ -95,7 +96,6 @@ def dataset(self, name):
         :param name: Name of the dataset.
 
         :rtype: :class:`gcloud.bigquery.dataset.Dataset`
-        :returns: a new ``Dataset`` instance
```

Comment:
Though redundant, we should view `:returns:` as "required". I didn't imagine 35697d49ea3a0ed76c565f7ecb68b4c8394ff196 would come from my tongue-in-cheek comment.